### PR TITLE
🚚(core) remove the concept of project in arnold

### PR DESCRIPTION
**Pull request by madmatah**
## Purpose

In k8s, the `Project` object type doesn't exist, it is called a `Namespace`.

## Proposal

All occurrences of project have been replaced with `namespace` to avoid confusion.

### CODE REVIEW

**Review by lunika — COMMENTED** [review]
There are references to `project` in the documentation. Did you plan to work on the documentation later ? Or to work on each part in the associated PR ?

**Comment by madmatah** [comment]
> There are references to `project` in the documentation. Did you plan to work on the documentation later ? Or to work on each part in the associated PR ?

@lunika  Yes, I deliberately left the documentation aside with the goal of rewriting it later.
This is a big work, some parts are deprecated, some other parts have already been rewritten in #582. So I think it will be more efficient to rewrite it in a dedicated PR.

File: bin/arnold [hunk 1]
```diff
@@ -182,34 +182,34 @@ OPTIONS:
 
 COMMANDS:
 
-> Prepare current project:
+> Prepare current environment:
```
Comment (jmaupetit) [inline bin/arnold]: I think there might be a confusion between what is called here an `environment` and the `env_type`. Does an `environment` means a customer/env_type combination to you?

If yes, I think we should still call it a namespace as it corresponds to a customer/env_type couple.

Comment (madmatah) [reply bin/arnold]: Yes, it means a customer/env_type combination to me.
Ok, I'll do what you recommend.

Comment (madmatah) [reply bin/arnold]: Done :heavy_check_mark: 
I also updated the commit message body.

**Review by lunika — APPROVED** [review]

**Pull request merged**